db/migrate/20210621104559_add_is_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsAdminToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :is_admin, :boolean, default: false
```

Comment:
How about adding `null: false` as well :).

Reply:
We can add index to it as well as per our conventions https://nimblehq.co/compass/development/code-conventions/ruby-on-rails/#relational-database

Reply:
True!! 
Updated, thanks!!!